src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -59,13 +59,17 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.generic_update_command('update', getter_name='iot_dps_get', setter_name='iot_dps_update',
                                  command_type=update_custom_util)
 
-    # iot dps access-policy commands
-    with self.command_group('iot dps access-policy', client_factory=iot_service_provisioning_factory) as g:
-        g.custom_command('list', 'iot_dps_access_policy_list')
-        g.custom_show_command('show', 'iot_dps_access_policy_get')
-        g.custom_command('create', 'iot_dps_access_policy_create', supports_no_wait=True)
-        g.custom_command('update', 'iot_dps_access_policy_update', supports_no_wait=True)
-        g.custom_command('delete', 'iot_dps_access_policy_delete', supports_no_wait=True)
+    # iot dps access-policy commands (Deprecated)
+    with self.command_group('iot dps access-policy',
+                            client_factory=iot_service_provisioning_factory,
+                            deprecate_info=self.deprecate(redirect='iot dps policy',
+                                                          expiration='2.35.0')
```

Comment:
The expiration causes lots of failures in CI after Azure CLI is updated to 2.35.0:

https://dev.azure.com/azure-sdk/public/_build/results?buildId=1475423&view=logs&j=168ccbe3-da49-5c0b-6478-08f7016e4bf5&t=10a595be-4223-5cc7-9a73-012871b6eaee

```
-  FAIL - HIGH severity: faulty_help_example_parameters_rule
    Help-Entry: `iot dps access-policy create` - 
	There is a violation:
	"az iot dps access-policy create --dps-name MyDps --resource-group MyResourceGroup --name MyPolicy --rights EnrollmentRead" is not a valid command.
	argument _subcommand: unknown parser 'access-policy' (choices: list, show, create, delete, update, linked-hub, certificate, policy).
...
-  FAIL - HIGH severity: expired_command_group
    Command-Group: `iot dps access-policy` - Deprecated command group is expired and should be removed.
...
-  FAIL - HIGH severity: missing_command_help
    Command: `iot dps access-policy create` - Missing help
    Command: `iot dps access-policy delete` - Missing help
    Command: `iot dps access-policy list` - Missing help
    Command: `iot dps access-policy show` - Missing help
    Command: `iot dps access-policy update` - Missing help

```